File: mvp_app/MyProject/app/src/main/java/com/tosmart/xiexuming/myapplication/MyApplication.java
package com.tosmart.xiexuming.myapplication;

import android.app.Application;

import com.tosmart.xiexuming.myapplication.reposity.DataRepository;

/**
 * @author xiexuming
 */
public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        DataRepository.getInstance(this);
    }

}

File: newpaserapp/app/src/main/java/com/example/newpaserapp/MyApplication.kt
package com.example.newpaserapp

import android.app.Application
import com.example.newpaserapp.repository.DataRepository

/**
 * @author xxm
 * 2022/7/21
 **/
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        DataRepository.getInstance(this)
    }
}